components/base/wheeled/sensorbase.go

```diff
@@ -80,16 +81,22 @@ func (sb *sensorBase) Reconfigure(ctx context.Context, deps resource.Dependencie
 		}
 	}
 
-	var oriMsName string
 	for _, ms := range sb.allSensors {
 		props, err := ms.Properties(context.Background(), nil)
 		if props.OrientationSupported && err == nil {
 			sb.orientation = ms
-			oriMsName = ms.Name().ShortName()
+			sb.logger.Infof("using sensor %s as orientation sensor for base", sb.orientation.Name().ShortName())
+		}
+	}
+
+	for _, ms := range sb.allSensors {
+		props, err := ms.Properties(context.Background(), nil)
+		if props.AngularVelocitySupported && props.LinearVelocitySupported && err == nil {
+			sb.velocitiesSensor = ms
+			sb.logger.Infof("using sensor %s as velocity sensor for base", sb.velocitiesSensor.Name().ShortName())
 		}
```

Comment:
We always pick the first available sensor that does not error as of now - it's just a choice we do with all lists of dependencies - we may break this into one sensor rather than a list of sensors at the end of this project.